tests/acceptance/features/bootstrap/OcisConfigContext.php

```diff
@@ -51,6 +52,28 @@ public function asyncUploadHasbeenEnabledWithDelayedPostProcessing(string $delay
 		);
 	}
 
+	/**
+	 * @Given :env has been enabled and set to :envValue
+	 *
+	 * @param string $env
+	 * @param string $envValue
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function envHasBeenEnabledAndSetTo(string $env, string $envValue): void {
+		$envs = [
+			$env => $envValue,
+		];
+
+		$response =  OcisConfigHelper::reConfigureOcis($envs);
+		Assert::assertEquals(
+			200,
+			$response->getStatusCode(),
+			"Failed to set OCIS_CORS_ALLOW_ORIGINS=" . $envValue
```

Comment:
```suggestion
			"Failed to set OCIS_SPACES_MAX_QUOTA=" . $envValue
```
Seems forgot to change here.